docs/docs/cloud/quick_start.md

```diff
@@ -160,10 +199,9 @@ This will start up the LangGraph API server locally. If this runs successfully,
 ```shell
 Ready!
 - API: http://localhost:8123
-2024-06-26 19:20:41,056:INFO:uvicorn.access 127.0.0.1:44138 - "GET /ok HTTP/1.1" 200
 ```
 
-You can now test this out! **Note: this local server is intended SOLELY for local testing purposes and is not performant enough for production applications, so please do not use it as such.** To test it out, you can go to another terminal window and run:
+You can now test this out! You can open another terminal tab/window and run:
```

Comment:
nit: you can do a quick curl to the OK endpoint as an easier step before issuing the full request -- it allows you to also show the user what they should see as the output of `/ok` -- and it's usefult o know about this endpoint for health / readiness checks

Reply:
sounds good -- will add